src/components/common/ChattingInput/ChattingInput.tsx

```diff
@@ -0,0 +1,80 @@
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+import type { InputHTMLAttributes, MouseEventHandler } from 'react';
+
+import { IconCamera } from '../Icons/IconCamera';
+import { Text } from '../Text';
+
+type ChattingInputProps = {
+  isChatAlive: boolean;
+  onClickSendButton: MouseEventHandler<HTMLButtonElement>;
+} & InputHTMLAttributes<HTMLInputElement>;
+
+export const ChattingInput = ({
+  isChatAlive,
+  value,
+  onClickSendButton,
+  ...props
+}: ChattingInputProps) => {
+  const sendButtonColor = isChatAlive ? (!!value ? 'secondary' : 'gray400') : 'gray500';
```

Comment:
```suggestion
  const sendButtonColor = isChatAlive ? (Boolean(value) ? 'secondary' : 'gray400') : 'gray500';
```

다음과 같이 한다면 DX가 향상될 것 같아요

Reply:
!!보단 Boolean이 형변환한다는 느낌이 강하네요. 알겠습ㄴ다